app/uk/gov/hmrc/helptosavestub/controllers/CitizenDetailsController.scala

```diff
@@ -63,6 +64,7 @@ object CitizenDetailsController extends BaseController {
       add3 <- initAdd(2)
       postcodeO <- address.last
```

Comment:
As discussed, almostAlways can return null/blank - for what we're doing at the moment, I think we need a forename and surname consistently returned.